config/test/kustomization.yaml

```diff
@@ -1,8 +0,0 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-bases:
-- ../default
-images:
-- name: controller
-  newName: etcd-cluster-operator
-  newTag: test
```

Comment:
This was used because `make deploy` uses `kustomize edit` which results in a modified kustomization file, containing the actual docker image names that have been built....and hence an annoying `git diff` because that file is under version control.
But actually I think that's okay. With the version based Docker image tags it makes it clear which version of the operator has been deployed and is being tested.
You just have to remember not to commit that file....but even if you do, it doesn't really matter because anyone who deploys the operator using our kustomize config will have to override the image names anyway.